lib/ansible/modules/web_infrastructure/ansible_tower/tower_project.py

```diff
@@ -208,21 +208,27 @@ def main():
         project = tower_cli.get_resource('project')
         try:
             if state == 'present':
-                org_res = tower_cli.get_resource('organization')
-                org = org_res.get(name=organization)
+                try:
+                    org_res = tower_cli.get_resource('organization')
+                    org = org_res.get(name=organization)
+                except (exc.NotFound) as excinfo:
+                    module.fail_json(msg='Failed to update project, organization not found: {0}'.format(excinfo), changed=False)
```

Comment:
Since we know that the message returned from Tower CLI isn't very helpful here, lets replace `excinfo` with `organization`. When I originally did this, I made a (bad) assumption that the Tower CLI error message would be useful.


    except exc.NotFound:
        module.fail_json(msg='Failed to update project, organization not found: {0}'.format(organization), changed=False)